packages/cojson/src/coValues/coList.ts

```diff
@@ -188,6 +192,13 @@ export class RawCoList<
   }
 
   processNewTransactions() {
+    if (this.isTimeTravelEntity()) {
+      throw new Error("Cannot process transactions on a time travel entity");
+    }
```

Comment:
This check is not necessary, because the new changes would be filtered out anyway.